04/global.css

```diff
@@ -1,2 +1,60 @@
 /*font-family: Georgia, Times, "Times New Roman", serif;
-font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
\ No newline at end of file
+font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;*/
+#header{
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 27px;
+    padding-left: 91px;
+    padding-top: 21px;
+    padding-bottom: 20px;
+    border-bottom: 1px;
+    border-bottom-color: orange;
+    border-bottom-style: solid;
+    border-bottom-width: thin;
+    font-weight: bolder;
+    letter-spacing: 3px;
+    color: orange;
+}
+
+#header_description {
+      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+            padding: 20px;
+    margin: 20px;
+    background-color: #F2F2F2;
+        letter-spacing: .5px;
+}
```

Comment:
Gotta watch that indentation–even in CSS :)

![](http://cl.ly/fRIH/Screen%20Shot%202016-03-17%20at%206.43.20%20AM.png)